gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10-bin.zip
```

Comment:
i'm not sure if i might have missed something elsewhere but this required some changes in our projects:

https://github.com/ministryofjustice/hmpps-template-kotlin/pull/217#issuecomment-2289046689

Reply:
>…downgrade until tested elsewhere…

well i did *some* testing in the linked project: using the JDK base image “worked”